docs/dev/cli/espresso-xcuitest/yaml-config.md

```diff
@@ -29,22 +29,38 @@ The below flags are required when writing your YAML file.
 
```

Comment:
Hey @FriggaHel -- this file is actually related to the old sauce runner CLI, not saucectl. we didn't delete it because we don't have an internal redirect process in place yet, but there is not need to update this file. I will add a deprecated badge to it and link to the saucectl docs.